Java-Loops-Control-Statement-Break.md

```diff
@@ -0,0 +1,57 @@
+# break
```

Comment:
A bit more descriptive headers are needed than just break. Maybe : Java : the **break** statement?